internal/util-collection/src/test/scala/SettingsTest.scala

```diff
@@ -135,8 +135,10 @@ object SettingsTest extends Properties("settings") {
     forAll(mkAttrKeys[Int](2)) {
       case List(key, derivedKey) =>
         val projectKeys = for { proj <- 1 to nrProjects } yield ScopedKey(Scope(1, proj), key)
-        val projectDerivedKeys = for { proj <- 1 to nrProjects } yield
-          ScopedKey(Scope(1, proj), derivedKey)
+        val projectDerivedKeys = for { proj <- 1 to nrProjects } yield ScopedKey(
+          Scope(1, proj),
+          derivedKey
+        )
```

Comment:
not very happy with these superfluous formatting changes every bump brings, but I guess we have no option.

Reply:
Agreed in general, though I actually think that these changes are good. I hope that since we are off an RC version that things will be more stable moving forward.